deployment/agent-deployment/ServiceGroupRoot/Parameters/ContainerInsights.Windows.Parameters.json

```diff
@@ -40,8 +40,8 @@
               "value": "__AGENT_IMAGE_TAG_SUFFIX__"
             },
             {
-                "name": "AGENT_IMAGE_FULL_PATH",
-                "value": "public/azuremonitor/containerinsights/__AGENT_RELEASE__:win-__AGENT_RELEASE____AGENT_IMAGE_TAG_SUFFIX__"
+              "name": "AGENT_IMAGE_FULL_PATH",
+              "value": "public/azuremonitor/containerinsights/__AGENT_RELEASE__:win-__AGENT_IMAGE_TAG_SUFFIX__"
```

Comment:
Lets confirm with Robbie about the contract we are going with for windows image and also preview images for the toggles.